### ISSUE TITLE
set all dates "disabled" except those marked dates 

### ISSUE THREAD

**Issue by kayazinc** [wix/react-native-calendars#60]
# Use-case
Possible to set all dates "disabled" except those marked dates configured?

Something similar to https://github.com/wix/react-native-calendars#date-marking, the differences are:
- all dates are disabled (will not response to user click)
- except those marked dates configured will be enabled.
```
<Calendar 
  // Collection of dates that have to be marked. Default = {}
  markedDates={{
    '2012-03-16': {marked: true},
    '2012-05-03': {marked: true},
    '2012-06-28': {disabled: true}
  }}
/>
```

**Comment by kcfgl** [comment]
I agree, this use-case is exactly what I need.  I just found this and am so excited to swap out my existing calendar component.  I need to be able to show a range of available dates, which means any other dates should be disabled.  Would be perfect solution.  Thanks again for this, looks amazing.

**Comment by tautvilas** [comment]
Hi I think this is a good idea and I hope I will add some option like `disabledByDefault=true` soon

**Comment by tautvilas** [comment]
Also PR would be welcome

**Comment by kcfgl** [comment]
@tautvilas Here is a first stab at a PR https://github.com/wix/react-native-calendars/pull/67

**Comment by tautvilas** [comment]
Solved in 1.8.1 with disabledByDefault prop
